File: Upcoder/Other/sum/LTCB-TH-02.cs
using System;
namespace Sum
{
    public class Program
    {
        static void Main(string[]args)
        {
            int n = int.Parse(Console.ReadLine());
            int tong = 0;
            int temp = 0;
            for (int i = 1; i <= n; i++)
            {
                temp = temp * 10 + 1;
                tong += temp;
            }
            
            Console.Write(tong);
        }
    }
}

File: Upcoder/Other/sum/LTCB-TH-02.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    cin >> n;
    int tong = 0;
    int temp = 0;
    while(n != 0)
    {
        temp = temp * 10 + 1;
        tong += temp;
        n --;
    }
    cout << tong;
}